src/App.tsx

```diff
@@ -31,65 +32,129 @@ export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  visibleGoods.sort((firstGood, secondGood) => {
+    if (sortType === SortType.ALPHABET) {
+      return firstGood.localeCompare(secondGood);
+    }
+
+    if (sortType === SortType.LENGTH) {
+      return Number(firstGood.length) - Number(secondGood.length);
```

Comment:
Maybe the `Number` is ​​redundant
```suggestion
      return firstGood.length - secondGood.length;
```